src/domain/event/formSections/responsibilitiesSection/ResponsibilitiesSection.tsx

```diff
@@ -91,13 +95,18 @@ const ResponsibilitiesSection: React.FC<ResponsibilitiesSectionProps> = ({
         <FieldColumn>
           <MultiLanguageField
             disabled={!isEditingAllowed}
-            labelKey={`event.form.labelProvider.${type}`}
+            labelKey={
+              !isExternalUser
+                ? `event.form.labelProvider.${type}`
+                : 'event.form.labelProvider.other'
+            }
```

Comment:
I missed this in my first review round.  
```
labelKey={
              isExternalUser
                ? 'event.form.labelProvider.other'
                : `event.form.labelProvider.${type}`
            }
```
could be more readable

Reply:
Fixed: [0b417dd](https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/185/commits/0b417dd72d690a923be1b4e90dbbbbc74d68d4fa)